src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,94 @@
+package lotto.domain;
+
+public class LottoResult {
+
+	private static final int FIRST_RANK_EARNING = 2000000000;
```

Comment:
들여쓰기에는 탭 문자가 사용되지 않습니다. 아래의 Google Java Style Guide를 참고해 주세요.

> Tab characters are not used for indentation.

```suggestion
    private static final int FIRST_RANK_EARNING = 2000000000;
```